core/benches/validation.rs

```diff
@@ -58,12 +61,12 @@ fn build_test_transaction(keys: KeyPair) -> Transaction {
 fn build_test_wsv(keys: KeyPair) -> WorldStateView<World> {
     WorldStateView::new({
         let mut domains = BTreeMap::new();
-        let mut domain = Domain::test(START_DOMAIN);
-        let account_id = AccountId::test(START_ACCOUNT, START_DOMAIN);
+        let mut domain = Domain::new(DomainId::new(START_DOMAIN).expect("is valid"));
```

Comment:
Everywhere outside this file you started using just `expect("Valid")`. May be you should also do it here for consistency?